internal/cli/pipeline_run.go

```diff
@@ -104,26 +104,47 @@ func (c *PipelineRunCommand) Run(args []string) int {
 		}
 
 		step.Update("Pipeline %q has started running. Attempting to read job stream sequentially in order", pipelineIdent)
+
+		steps := len(resp.JobMap)
+		step.Update("%d steps detected, run sequence %d", steps, resp.Sequence)
```

Comment:
We should make this its own `ui.Output` since I don't think we intend to update this as we execute a pipeline. And I would like to preserve the message on 106 too

Reply:
Yeah, I struggled with that back and forth. I liked that as a step update, it would turn red if the run wasn't successfully completed, & didn't like how the UI output looked, but yea definitely should preserve 106.

Reply:
I recommend making this its own step group! Then we can still get the updates as the pipeline runs as well as the red X for failures